portal-api/api-tests/deal-versioning.api-test.js

```diff
@@ -49,6 +49,7 @@ const generateVersion0ApplicationToSubmit = () => ({
  */
 const generateVersion0ApplicationResponse = (makerId) => ({
   ...generateVersion0ApplicationToSubmit(),
+  mandatoryVersionId: expect.any(Number),
```

Comment:
👀?

Reply:
The changes I've made have clearly re-ordered the tests a bit & one of them deletes the mandatory criteria. I'll do this properly & add in the mandatory criteria in a beforeAll